src/features/logging/shared/log.js

```diff
@@ -0,0 +1,22 @@
+import { globalScope } from '../../../common/constants/runtime'
+import { cleanURL } from '../../../common/url/clean-url'
+
+export class Log {
+  /**
+   *
+   * @param {number} timestamp - Unix timestamp
+   * @param {string} message - message string
+   * @param {{[key: string]: *}} attributes - other attributes
+   * @param {enum} level - Log level
+   */
+  constructor (timestamp, message, attributes, level) {
+    /** @type {long} */
+    this.timestamp = timestamp
+    this.message = message
+    this.attributes = attributes
+    this.logType = level
+    this.session = {
```

Comment:
Can we improve this naming?

Reply:
sure, but can you start by divulging what exactly is unclear here?

Reply:
"session" should be a restricted keyword at this point because there are many things tied to sessions and we have actual objects/variables that are named closely so, like `session` from the session-entity which is also on runtime.session.
It's unclear what `this.session` here signifies or why it's needed instead of simply `this.url` (what is this.url supposed to be too?) -- not that either of them appear to be referenced at all in this PR

Reply:
Ah i see your confusion here. -- this object is essentially the individual log Event, the properties here are what will show up for the DB item that is saved, 1:1.  The attributes here are the agreed-upon attributes that are supposed to constitute the shape of the log item.  In other words, the attribute `session.id` will be a queryable property in NR1.

Reply:
i'll add some comments to that file to make it more explicit